src/style/blocks/CardList.css

```diff
@@ -0,0 +1,88 @@
+@import url(./StarsBlock.css);
+@import url(./../variables.css);
+
+.CardList {
```

Comment:
Why do you name start with a capital letter?

Reply:
> Why do you name start with a capital letter?
I`m try to use React style bem naming convention